tests/cxx/explicit_instantiation2-template_helpers.h

```diff
@@ -0,0 +1,40 @@
+//=== explicit_instantiation2-template_helpers.h - test input file for iwyu ==//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+#ifndef INCLUDE_WHAT_YOU_USE_TESTS_CXX_EXPLICIT_INSTANTIATION2_TEMPLATE_HELPERS_H_
+#define INCLUDE_WHAT_YOU_USE_TESTS_CXX_EXPLICIT_INSTANTIATION2_TEMPLATE_HELPERS_H_
+
+template <class T>
```

Comment:
Is this a re-declaration of the other `Template` in explicit_instantiation-template.h? In that case, maybe use `struct` instead of `class`, so they're consistent -- some compilers warn when decls differ only by class/struct keyword.

If not, please choose a unique-r name.